src/utils/quick-open.js

```diff
@@ -50,14 +51,16 @@ export function parseLineColumn(query: string) {
   }
 }
 
-export function formatSourcesForList(source: RelativeSource) {
+export function formatSourcesForList(source: RelativeSource, tabs: TabList) {
   const title = getFilename(source);
   const subtitle = endTruncateStr(source.relativeUrl, 100);
-
   return {
     value: source.relativeUrl,
     title,
     subtitle,
+    icon: tabs.includes(source.url)
+      ? "tab result-item-icon"
+      : getSourceClassnames(source, "result-item-icon"),
```

Comment:
we can do classnames("result-item-icon",  getSourceClassnames(source))` to keep the util simpler

Reply:
my thought process was that this would typically be used on an image tag, so it's highly likely you'd want to add additional classes that are relevant to the source class name but not derived from the extension type. 

This does expand on its defined behavior maybe a bit too much.